src/mlagility/cli.py

```diff
@@ -355,7 +355,7 @@ def main():
     # we alter argv to insert the command for them.
 
     if len(sys.argv) > 1:
-        if sys.argv[1] not in subparsers.choices.keys() and sys.argv[1].endswith(".py"):
+        if sys.argv[1] not in subparsers.choices.keys() and ".py" in sys.argv[1]:
```

Comment:
Isn't this a less robust solution than `endswith`?

Reply:
Will do something like `name.split(":")[0].endswith(".py")`

Reply:
Added a nice decode function that I can call in cli.py and use with endswith()